dashboard/app/views/pd/workshop_mailer/teacher_cancel_receipt.html.haml

```diff
@@ -0,0 +1,28 @@
+:css
+  body {
+    font-family: Gotham, sans-serif;
+  }
+
+%p
+  You have canceled your registration for the Code.org
+  = @workshop.course
+  = @workshop.subject
```

Comment:
I think the spec had a typo and it's meant to be <course> <subject> **workshop** with <organzier name>